2024-02-28 ARIMA Forecasting in R/ARIMA Forecasting in R.Rmd

```diff
@@ -0,0 +1,264 @@
+---
+title: |
+  ![](BSA_report_header.jpg){width=100%}  
+    
+  ARIMA Forecasting in R
+#author: ""
+#date: "2024-28-02"
+
+output: 
+  html_document:
+    css: "style/style.css"
+    toc: true
+    toc_depth: 2
+    number_sections: true
+    toc_float: 
+      collapsed: false
+editor_options: 
+  chunk_output_type: inline
+---
+
+<style type="text/css">
+
+body, td {
+   font-size: 16px;
+   font-family: sans-serif;
+}
+</style>
+<html lang="en">
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, warning = FALSE, message = FALSE)
+```
+
+In this session we will cover using ARIMA models (Autoregressive integrated moving average models) with time series data in R. This session will give you insight in to where you might want to use these types of models and why, but importantly it will also cover where these models are not suitable. We will then cover how to fit models to your data sets and check their validity. Finally we will cover forecasting your time series so you can make predictions on the future of your data.
+
+Please be aware that ARIMA models are not the only type of models out there for forecasting your time series data, there are others such as exponential smoothing models and time series regression models. These other models will not be covered in today's session.
+
+This session will follow the process laid out in the book [Forecasting: Principals and Practice](https://otexts.com/fpp3/arima.html). This book goes in to a great amount of detail on modelling and forecasting and I cannot reccomend it enough as a great place to get further information on everything covered in today's session.
+
+# Load Packages and Data
+
+We will install the following packages for this session. fpp3 is the forecasting package we will use in this session, loading this package in will automatically load in other useful packages such as dplyr and ggplot2 as well as tsibble and fable which are the packages used for applying the ARIMA models. Please note there are many other packages available to help you with ARIMA modelling and forecasting such as the forecast package and the astsa package to name a few.
```

Comment:
Consider wrapping package names with the ` character. This highlight them more obviously as packages